webdriver-spec.html

```diff
@@ -6030,7 +6030,7 @@
         </tr>
       </table>
 
-    <p>A <dfn>tick</dfn> is the basic unit of time over which actions
+    <p>A <dfn data-lt="tick|ticks">tick</dfn> is the basic unit of time over which actions
```

Comment:
Ah I see, I used as example `<dt><dfn data-lt="remote end|remote ends|remote">Remote End</dfn>` https://github.com/w3c/webdriver/blob/master/webdriver-spec.html#L541 

@andreastt in that case is the `remote end` in `data-lt` is due to upper/lower case?